yarn.lock

```diff
@@ -10464,6 +10477,15 @@ cross-spawn@^6.0.0, cross-spawn@^6.0.5:
     shebang-command "^1.2.0"
     which "^1.2.9"
 
+cross-spawn@^7.0.0, cross-spawn@^7.0.2:
```

Comment:
Can combine this with `cross-spawn@7.0.1`

Reply:
Unfortunately not without fixing it to `7.0.1`, as the other reference is hardcoded to use that version specifically. Do you think that's decreeable? It's coming from `react-dev-utils` which for some reason uses pinned versions in their `package.json`, so it's not up to us 😞

Reply:
I don't see a valid reason for them pinning the dependendencies to the patch versions. 7.0.2 and 7.0.3 only contain bug fixes, however that is not to say it wouldn't cause issues down the road.

Overall, I am +1 here to combine cross-spawn to reduce our surface area.

Reply:
I'm not saying they have a good reason, just that it means we either can't use a newer version (lowest common denominator), or that we have to use the resolutions field in our package.json if we want to unify them all.

I'm normally not in favor of using the resolutions field unless absolutely necessary as it means we have to maintain it, so I'm currently leaning towards leaving this as is.

Reply:
I see, apologies for not catching that initially. I agree, it's not justified to add a resolution field for this. I will follow-up with a PR/issue in react-dev-utils.